### PR TITLE
Support file upload.

### PR DESCRIPTION
**Pull request by justinlittman**
refs #409

image

### CODE REVIEW

File: libsys_airflow/dags/vendor/data_fetcher.py [hunk 1]
```diff
@@ -75,7 +75,6 @@ def setup():
         logger.info(f"Params are {params}")
 
         os.makedirs(params["download_path"], exist_ok=True)
-        os.makedirs(params["archive_path"], exist_ok=True)
```
Comment (justinlittman) [inline libsys_airflow/dags/vendor/data_fetcher.py:78]: This moved into the archive function.

File: libsys_airflow/dags/vendor/default_data_processor.py [hunk 2]
```diff
@@ -61,12 +62,9 @@ def setup():
 
         pg_hook = PostgresHook("vendor_loads")
         with Session(pg_hook.get_sqlalchemy_engine()) as session:
-            vendor_interface = session.scalars(
-                select(VendorInterface).where(
-                    VendorInterface.folio_interface_uuid
-                    == params["vendor_interface_uuid"]
-                )
-            ).first()
+            vendor_interface = load_vendor_interface(
```
Comment (justinlittman) [inline libsys_airflow/dags/vendor/default_data_processor.py:65]: Loading VendorInterface moved  into shared function here and below.

File: libsys_airflow/plugins/vendor/models.py [hunk 3]
```diff
@@ -120,6 +126,7 @@ class FileStatus(enum.Enum):
     not_fetched = "not_fetched"
     fetching_error = "fetching_error"
     fetched = "fetched"
+    uploaded = "uploaded"
```
Comment (justinlittman) [inline libsys_airflow/plugins/vendor/models.py:129]: Note the new status.

File: libsys_airflow/plugins/vendor_app/templates/vendors/vendor.html [hunk 4]
```diff
@@ -48,6 +48,11 @@
 </script>
 <h1>{{ vendor.display_name }}</h1>
 
+<dl class="dl-horizontal">
+    <dt>Organization ID</dt>
+    <dd>{{ vendor.folio_organization_uuid }}</dd>
```
Comment (justinlittman) [inline libsys_airflow/plugins/vendor_app/templates/vendors/vendor.html:53]: It is helpful to have this displayed.

**Review by aaron-collier — APPROVED** [review]

**Pull request merged**